OKR.md

```diff
@@ -33,4 +34,5 @@ Quarterly statement of OKRs (Objectives and Key Results) for the "IPFS in Web Br
 **Structure documentation and discussion around the primary Browser vendor concerns:**
 
 - `dweb:` proposal is documented
```

Comment:
FYSA @lgierth created an outline for this: https://github.com/ipfs/specs/pull/167,